[2024-11-26T01:51:00.076+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-26T01:51:00.099+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T01:50:54.383803+00:00 [queued]>
[2024-11-26T01:51:00.111+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T01:50:54.383803+00:00 [queued]>
[2024-11-26T01:51:00.112+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-26T01:51:00.129+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ingest_performance_reports_task> on 2024-11-26 01:50:54.383803+00:00
[2024-11-26T01:51:00.140+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7365) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-26T01:51:00.143+0000] {standard_task_runner.py:72} INFO - Started process 7372 to run task
[2024-11-26T01:51:00.141+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ETL_quarterly_performance_reports', 'ingest_performance_reports_task', 'manual__2024-11-26T01:50:54.383803+00:00', '--job-id', '318', '--raw', '--subdir', 'DAGS_FOLDER/project_dag.py', '--cfg-path', '/tmp/tmphc29noa3']
[2024-11-26T01:51:00.144+0000] {standard_task_runner.py:105} INFO - Job 318: Subtask ingest_performance_reports_task
[2024-11-26T01:51:00.201+0000] {task_command.py:467} INFO - Running <TaskInstance: ETL_quarterly_performance_reports.ingest_performance_reports_task manual__2024-11-26T01:50:54.383803+00:00 [running]> on host 6a246d8d590f
[2024-11-26T01:51:00.675+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_quarterly_performance_reports' AIRFLOW_CTX_TASK_ID='ingest_performance_reports_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-26T01:50:54.383803+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-26T01:50:54.383803+00:00'
[2024-11-26T01:51:00.677+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-26T01:51:01.390+0000] {logging_mixin.py:190} INFO - Error: column "time_period_amount" of relation "amountdescriptions" does not exist
LINE 1: ... AmountDescriptions (amount_desc_id, description, time_perio...
                                                             ^
[2024-11-26T01:51:02.913+0000] {logging_mixin.py:190} INFO - Error: column "benefit_type" of relation "activities" does not exist
LINE 1: ...end_date, national_objective, responsible_entity, benefit_ty...
                                                             ^
[2024-11-26T01:51:04.364+0000] {logging_mixin.py:190} INFO - Error: column "grantee_activity_number" of relation "performancereports" does not exist
LINE 1: INSERT INTO PerformanceReports (activity_id, Grantee_Activit...
                                                     ^
[2024-11-26T01:51:04.365+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-26T01:51:04.373+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-26T01:51:04.374+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ETL_quarterly_performance_reports, task_id=ingest_performance_reports_task, run_id=manual__2024-11-26T01:50:54.383803+00:00, execution_date=20241126T015054, start_date=20241126T015100, end_date=20241126T015104
[2024-11-26T01:51:04.416+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-26T01:51:04.435+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-26T01:51:04.438+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
